### DISCUSSION TITLE
Ideas

### DISCUSSION THREAD

**Discussion by gtRobCodes** [andycb/AdventurerClientJS#267] [Ideas]
Looks like this was build more for an FDM printer, however it works with my FF Foto 8.9...  I was curious if you could have it accept svgx files and also allow you to drop files more like a folder on to the printer's internal memory to print later from the device.  Example: I want to send 5 different prints to the printer in another room, then as they finish, I can pull them and go to the internal memory from the screen and start the next.  Removes keeping track of a USB drive.

**Comment by andycb** [comment]
Hey, I think must FlashForge printers use the same protocol, so this tool will probably work to some degree with all of them. So its great that it works with the Foto too (I looked that printer up when you mentioned it, and now I kind of want it, how is it).

 The tool doesn't do any translations of the file as it transfers it, so if the printer can natively accept a svgx file from USB, then this should work - and is pretty easy to change. 

Just transferring the file(s) without printing would be pretty simple - the app actually does that already - transfers the file, then selects it to print. A new page could be added to do this in the app. Right now, I'm working on some other projects so I don't know when I'll get a changes to look at this myself. You are more than welcome to take a look at those changes if you want.
